app/views/sessions/new.html.erb

```diff
@@ -22,15 +22,13 @@
   </div>
 <% end %>
 
+<% @preferred_auth_provider='microsoft' %>
```

Comment:
This looks like it could be unintended?

Reply:
Yeah, just left over from testing. On it.

Reply:
No problem. Please also avoid fixup commits, see https://github.com/openstreetmap/openstreetmap-website/blob/master/CONTRIBUTING.md#pull-requests

Reply:
Resolved with new commit

Reply:
Squashed the commits into one and addressed some linting issues in my code. Could you please approve the workflow to run the necessary checks?